translations/de/info/task_description.html

```diff
@@ -0,0 +1,59 @@
+<div class="story">
+    <div style="text-align: center">
+        <img src="{{MEDIA}}robots.png" style="max-height: 350px">
+    </div>
+    <p>
+        Mit 50 verschiedenen Steckertypen findet man keine passenden Geräte oder diese würden sehr teuer sein.
+        Sobald aber ein elektrischer Anschluss standardisiert wird, können viele Firmen Geräte designen,
+        Wettbewerb entsteht, mit wachsender Auswahl und besseren Preisen für Konsumenten.
+        <br>
+        -- Bill Gates
+
+    </p>
+
+</div>
+
+<p>
+    Du erhältst eine nicht-leere Liste von Ganzzahlen (X).
+    Für diese Aufgabe, solltest Du eine Liste bestehend nur aus den nicht-einmaligen Elementen daraus zurückgeben.
```

Comment:
to: _In dieser Aufgabe sollst Du eine Liste nur aus den nicht-einmaligen Elementen daraus zurückgeben._